test/functional/helpers.py

```diff
@@ -0,0 +1,142 @@
+# taken from https://github.com/mozilla-b2g/gaia/blob/master/tests/python/gaia-ui-tests/gaiatest/gaia_test.py#L858
+# XXX factor out into utility object for use by other tests
+from marionette_driver.by import By
+from marionette_driver.errors import NoSuchElementException, StaleElementException
+# noinspection PyUnresolvedReferences
+from marionette_driver import Wait
+
+import pyperclip
+import re
+import urlparse
+import time
+
+from serversetup import ROOMS_WEB_APP_URL_BASE
+from config import USE_LOCAL_STANDALONE
+
+
+def wait_for_element_displayed(self, by, locator, timeout=None):
```

Comment:
Oh, and this also means that we should be able to maintain the existing:

```python
self.switch_to_panel()
```

style of calling the functions.